README.md

```diff
@@ -12,7 +12,19 @@
 The HistFitter Group mailing-list, for **any** of your questions: <atlas-phys-susy-histfitter@cern.ch>
 
 ## Acquire
-HistFitter has 2 stable versions, one complient with Python2 (v0.66.0) and one with Python3 (v1.x.y).
+
+### StatAnalysis
+
+HistFitter is available in the ATLAS StatAnalysis release, which can be loaded with:
+
+```
+setupATLAS
```

Comment:
These are ATLAS specific instructions. I think we can keep this generic and write ATLAS specific things on the internal twiki.

Reply:
I see, you're moving this up from below.  I would still keep the generic instructions first.  Then put this after.

Reply:
That makes sense, let me move it around then.